charts/dvpe-deployment-gloo/values.yaml

```diff
@@ -8,7 +8,7 @@ externalSecrets:
     # externalSecrets.service.key -- `Key` to AWS Secret Manager object where all sensitive application data should be stored. Each key in the Secret Manager Object should be named like your needed environment variable
     key:
   oidc:
-    # externalSecrets.oidc.key -- `Key` to AWS Secret Manager object where the client secret for OIDC provider should be stored. The key in the Secret Manager Object has to be named as the given `gloo.authConfig.spec.configs.oauth.client_id`. The value has to be formated as `clientSecret: <secret>`.
+    # externalSecrets.oidc.key -- `Key` to AWS Secret Manager object where the client secret for OIDC provider should be stored. The key in the Secret Manager Object has to be named as the given `gloo.authConfig.spec.configs.oauth.client_id`. The value has to be formatted as `clientSecret: <secret>`.
```

Comment:
Same here, change `oidc` to `oauth2`.

With adding the clientCredentials Flow it is not longer oidc

Reply:
done